endpoints-control/src/main/java/com/google/api/control/model/KnownMetrics.java

```diff
@@ -110,12 +86,18 @@
   private MetricKind kind;
   private ValueType type;
   private Update updater;
+  private Mark mark;
 
   private KnownMetrics(String name, MetricKind kind, ValueType type, Update updater) {
+    this(name, kind, type, updater, Mark.PRODUCER);
```

Comment:
I think perhaps better a consumer metric appear in a producer graph than not have a producer metric show up. One could argue for no default I suppose.